src/enr/enr.ts

```diff
@@ -124,6 +124,10 @@ export class ENR extends Map<ENRKey, ENRValue> {
   }
   set multiaddrUDP(multiaddr: Multiaddr | undefined) {
     if (!multiaddr) {
+      this.set("ip", (undefined as unknown) as Buffer);
+      this.set("udp", (undefined as unknown) as Buffer);
+      this.set("ip6", (undefined as unknown) as Buffer);
+      this.set("udp6", (undefined as unknown) as Buffer);
```

Comment:
Is there a type-safe way of achieving the same result? Maybe by setting the IP to an empty buffer and checking by length?

Reply:
yeah setting to empty buffer should be good for type safety, @wemeetagain please confirm ..

Reply:
we can do `this.delete(key)`